packages/widget-next-playground/src/app/components/DrawerControls/DesignControls/DesignControls.style.tsx

```diff
@@ -0,0 +1,204 @@
+import { autocompletePopperZIndex } from '../DrawerControls.style';
+
+export const TabButtonsContainer = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  backgroundColor: getCardFieldsetBackgroundColor(theme),
+  borderRadius: Math.max(
+    theme.shape.borderRadius,
+    theme.shape.borderRadiusSecondary,
+  ),
+  padding: theme.spacing(0.5),
+  gap: theme.spacing(0.5),
+  height: '3.5rem',
+}));
+
+const controlSelected = (theme: Theme) => ({
+  backgroundColor:
+    theme.palette.mode === 'dark'
+      ? theme.palette.background.default
+      : theme.palette.common.white,
+  borderRadius: theme.shape.borderRadiusSecondary,
+  boxShadow: `0px 2px 4px ${alpha(theme.palette.common.black, 0.04)}`,
+});
+
+interface TabButtonProps {
+  selected?: boolean;
+}
+export const TabButton = styled(ButtonBase)<TabButtonProps>(({
+  theme,
+  selected,
+}) => {
+  const selectedStyle = selected
+    ? {
+        ...controlSelected(theme),
+      }
+    : {};
+
+  return {
+    height: '100%',
+    width: '100%',
+    fontSize: '1rem',
+    fontWeight: 700,
+    ...selectedStyle,
+  };
+});
+
+export const TabCustomInput = styled(InputBase)<TabButtonProps>(({
+  theme,
+  selected,
+}) => {
+  const selectedStyle = selected
+    ? {
+        '&:not(:focus)': {
+          ...controlSelected(theme),
+        },
+      }
+    : {};
+
+  return {
+    height: '100%',
+    width: '100%',
+
+    [`.${inputBaseClasses.input}`]: {
+      height: '100%',
+      width: '100%',
+      padding: 0,
+      textAlign: 'center',
+      '&::placeholder': {
+        fontSize: '1rem',
+        fontWeight: 700,
+        opacity: 1,
+      },
+      '&:focus': {
+        ...controlSelected(theme),
+      },
+      ...selectedStyle,
+    },
+  };
+});
+
+export const ColorSwatches = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  gap: theme.spacing(1),
+}));
+
+interface ColorSwatchProps {
+  color: string;
+}
+export const ColorSwatch = styled(
+  (props: BoxProps) => <Box {...props}>&nbsp;</Box>,
+  {
+    shouldForwardProp: (prop) => prop !== 'color',
+  },
+)<ColorSwatchProps>(({ theme, color }) => ({
+  width: theme.spacing(3),
+  height: 'auto',
+  backgroundColor: color,
+  content: '" "',
+}));
+
+export const ColorSelectorContainer = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  backgroundColor: getCardFieldsetBackgroundColor(theme),
+  borderRadius: Math.max(
+    theme.shape.borderRadius,
+    theme.shape.borderRadiusSecondary,
+  ),
+  padding: theme.spacing(0.5),
+  paddingLeft: theme.spacing(2.5),
+  gap: theme.spacing(0.5),
+  height: '3.5rem',
+  textTransform: 'capitalize',
+}));
+
+export const ColorInput = styled(InputBase)<InputBaseProps>(
+  ({ theme, value }) => ({
+    position: 'relative',
+    border: 'none',
+    height: '100%',
+    width: 97,
+    padding: 0,
+    backgroundColor: value as string,
+    borderRadius: theme.shape.borderRadiusSecondary,
```

Comment:
Should we apply the same logic as we have for Tabs to make the radius match the container radius proportionally?

```suggestion
    borderRadius: Math.max(theme.shape.borderRadius, theme.shape.borderRadiusSecondary) - 4,
```

### Before

<img width="362" alt="image" src="https://github.com/lifinance/widget/assets/18644653/97460d62-4bf5-4185-901f-c4bdca7269d4">

### After

<img width="342" alt="image" src="https://github.com/lifinance/widget/assets/18644653/cfd31c8b-ff3e-4716-9956-0efc5114bd3b">